### DISCUSSION TITLE
Does anyone know if there is any incompatibility between python-socketio (server) and iOS?

### DISCUSSION THREAD

**Discussion by euhake** [miguelgrinberg/python-socketio#1295] [Q&A]
I have a websocket server written in python-socketio, and my friend is building an application in react-native using expo, on Android it connects to websocket without problems, however, on iOS it does not connect at all, nor does it provide any type of connection error or log, both on the server and on the client.

I've looked everywhere to see if there is some kind of incompatibility, but I haven't been successful.

**Comment by miguelgrinberg** [comment]
Never heard of any issues with iOS. The Socket.IO protocol uses HTTP and WebSocket requests, all fairly standard stuff that has been available in browsers for many years.
